src/envoy/http/jwt_auth/token_extractor.cc

```diff
@@ -82,9 +82,18 @@ void JwtTokenExtractor::Extract(
   for (const auto& header_it : header_maps_) {
     const HeaderEntry* entry = headers.get(header_it.first);
     if (entry) {
+      std::string token;
```

Comment:
Some comment why we want to do this would be good. May also create an issue to make the same change in upstream JWT ?

Reply:
it's added below ```// If the header value has prefix, trim the prefix.```

Reply:
I mean it's unclear what does 'prefix' mean without context.